dashboard_schema.json

```diff
@@ -312,6 +312,71 @@
               }
             }
           },
+          // "gauge" widget type.
+          {
```

Comment:
@juliusv need your help on this, when i try to save the validation error being returned says only how the gauge doesn't conform to chart/pie/frame, it makes no mention of it not conforming to gauge.